packages/react-ui-map/src/Mapbox/index.js

```diff
@@ -51,14 +47,7 @@ export default class Mapbox extends PureComponent {
     })
 
     map.on('load', () => {
-      if (this.state.map !== map) {
-        this.setState(map)
-      }
-
-      sources.forEach((v, i) => {
-        map.addSource(sources[i].id, sources[i].source)
-        map.addLayer(layers[i].layer)
-      })
+      this.setState({ map })
```

Comment:
Does this need to be set on load here or can it be set directly after the instantiation for `new mapboxgl.Map()`?